dbg.h

```diff
@@ -626,6 +649,19 @@ class DebugOutput {
     return std::forward<T>(value);
   }
 
+  timestamp::timestamp_t&& print(const std::string& type, timestamp::timestamp_t value) const {
```

Comment:
Instead of repeating the whole printing logic here, we could write an overload for `pretty_print`.

Reply:
A specialized `pretty_print` works neat and elegant indeed and this is how this project works, but this would prevent me from using `ANSI_XXX`s. I just think printing timestamp with color might be better, but I can to forget about it if unnecessary.